modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/search/bar/portlet/display/context/factory/SearchBarPortletDisplayContextFactoryTest.java

```diff
@@ -544,6 +571,19 @@ private void _setUpThemeDisplay() {
 		);
 	}
 
+	private void _stubSearchBarPortletDisplayContextFactory(
```

Comment:
I would rename that method to `_setUpSearchBarPortletDisplayContextFactory()` or `_mockSearchBarPortletDisplayContextFactory()`

@felipelorenz @AlmirFe

Reply:
And would create the inside `SearchBarPortletDisplayContextFactory` the method

Reply:
Better than that is use the method that already exists to add these changes, you could add that in `_createSearchBarPortletDisplayContextFactory(x, x, x);`

Reply:
That change will be cleaner than add a new methods

Reply:
That change will be cleaner than add a new methods

Reply:
@felipelorenz @AlmirFe let me know if you have some doubt about that

Reply:
Open a new Pull Request #518